test/unittest/tests/block/test_bitstring.cpp

```diff
@@ -6,9 +6,10 @@
  * See the file LICENSE for redistribution information.
  */
 
-#include "wt_internal.h"
 #include <catch2/catch.hpp>
 
+#include "wt_internal.h"
+
 TEST_CASE("Bitstring macros: __bit_byte", "[bitstring]")
```

Comment:
Can you please fix the `bitVectorSize` and `bitVector` variables too?

Reply:
ac72d1a85